### ISSUE TITLE
Street lighting overlay visibility issue

### ISSUE THREAD

**Issue by thefirethirteen** [streetcomplete/StreetComplete#4631]
**How to Reproduce**
1. Hamburger menu > Overlays > Street lighting > OK
2. Choose any street that is red.

**Expected Behavior**
The user should be able to clearly distinguish the selected portion without street lighting data from other portions of the same street which also don't have street lighting data.

**Versions affected**
StreetComplete v49.0
Android 11

**Screenshots**
A portion that is NOT selected.
![Screenshot_20221114-181332_1.png](https://user-images.githubusercontent.com/78559591/201713651-d4cb24ab-ba9b-46a9-b8f8-ce5611b3636d.png)

The same portion, but is selected.
![Screenshot_20221114-181340_1.png](https://user-images.githubusercontent.com/78559591/201713940-3563722a-06bd-41bf-9f1a-f3bb3638abd6.png)

The same portion, selected, zoomed out.
![Screenshot_20221114-181353_1.png](https://user-images.githubusercontent.com/78559591/201714027-b66c29a5-bbd4-4879-bb6a-48cb2e274a07.png)

Another street that is lit and selected, showing how easily you should be able to distinguish a non-selected portion from a selected one.
![Screenshot_20221114-181459_1.png](https://user-images.githubusercontent.com/78559591/201714214-ce44ee06-b338-44d7-977b-0473dcce117b.png)

**Comment by thefirethirteen** [comment]
It seems I've lost the 'issue' part of my issue.
Tl;dr: When selecting a portion of a road without lighting data, we get red on red, and you can't really tell what you've selected (in both light and dark mode).

**Comment by legofahrrad** [comment]
Duplicate of #4417
